src/utils/room.js

```diff
@@ -0,0 +1,25 @@
+// Note: although it's possible to extract this from the RoomClass enum,
+// it would need to be compiled first, which is only done in schema.js
+export const validRooms = [
+  'Living room',
+  'Kitchen',
+  'Dining',
+  'Bedroom',
+  'Kids bedroom',
+  'Bathroom',
+  'Nursery',
+  'Recreation',
+  'Office',
+  'Gym',
+  'Hallway',
+  'Toilet',
+  'Front door',
+  'Garage',
+  'Terrace',
+  'Garden',
+  'Driveway',
+  'Carport',
+  'Other',
+];
+
+export const isValid = room => validRooms.includes(room);
```

Comment:
A `Set` instance would do beautifully here. `isValid` could check `validRooms.has(room)`.

Reply:
(This is part of my devious plot to raise global awareness for `Set`)

Reply:
Oh yeah! Set is a thing!Consider total global awareness raised by 1 person.